Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'pry', '~> 0.10', group: :development, require: false
 gem 'aruba', '~> 0.7.4', require: false
 gem 'rspec', '~> 3.0', require: false
 gem 'cucumber', '~> 2.0', require: false
+gem 'capybara', require: false # middleman-core forces all plugins to declare this
```

Comment:
We chatted about this in an issue a few months ago. core adds an unnecessary capybara dep just in case somebody might want to use it, which forces everybody to declare it.